CI/build-mudlet-for-windows.sh

```diff
@@ -159,15 +159,6 @@ fi
 echo " ... qmake done."
 echo ""
 
-export WITH_CCACHE="YES"
```

Comment:
This workaround is no longer needed since ccache is detected and used automatically now.